linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/tests/correct_stig_noninteractive.pass.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
It is a coincidence that the default value of `var_accounts_maximum_age_login_defs` and the value selected by STIG profile are the same.

```suggestion
# variables = var_accounts_maximum_age_login_defs=60
```